app/models/work.rb

```diff
@@ -0,0 +1,43 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+
+  validates :category, presence: true
+
+  validates :title, presence: true
+  validates :title, uniqueness: true
+
+  validates :title, length: { in: 1..35 }
+
+
+  def self.works_with_vote_tallies
```

Comment:
These methods seems overly complicated.  Couldn't you do a method to get the works in order by the number of votes with:

```ruby
works = Work.all.to_a
return works.sort_by { |work| -work.total_votes }
```
And you can always get the vote tally for a work with:  `work.votes.count`